news/2025/2025-03-09-gmt-apps-now-open.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: Global Moderation Team applications now open!
+date: 2025-03-09 12:00:00 +0000
+---
+
+If you have a burning passion towards the osu! community, and have great interest in keeping osu! safe for everyone, this is your time to shine.
+
+![](/wiki/shared/news/banners/gmt-generic.jpg)
+
+The [Global Moderation Team](/wiki/People/Global_Moderation_Team) has been a part of the game in some shape or form from its inception. If you are not aware of what we do, the team is responsible for overseeing the entire osu! platform, as well as assist community members with any queries, regardless of the nature.
+
+The team's reach is far and wide. From chats, comments, forums, to various community projects and the mapping community at large, we are always looking to expand our reach and make sure that everyone feels safe and welcome, and that any inappropriate activities get properly dealt with.
+
+If this sounds like something you want to be a part of, we want you to apply!
+
+### Application criteria
+
+Before your scroll down to the bottom of the news post and press that apply link, we must go through a couple of criteria relating to your soon-to-be submitted application:
+
+- **You must be at least 18 years old. This is [NOT negotiable](https://osu.ppy.sh/legal/Terms#your-responsibilities-in-volunteer-positions).** As part of the Global Moderation Team, you will come in contact with inappropriate content that's not suitable for everyone.
+- **Your account must be at least one year old.** This is to make sure that you accrued some tenure within the osu! community before applying.
+- You must be active within the osu! community. We expect prospective moderators to be active within the communities they are interested in moderating. While metrics are not the be-all and end-all when evaluating your applications, there is a level of activity that's expected.
+- If selected, you will need to adhere to the [Contributor Covenant Code of Conduct](/wiki/Rules/Contributor_Code_of_Conduct).
+
+As always, we will look through every single application, but if you fit into the following categories, we'll count that as a plus:
+
+- you speak Turkish on a near-native level, being active in `#turkish` is a plus
+- you speak Ukrainian on a near-native level, being active in `#ukrainian` is a plus
+- you speak Portuguese on a near-native level, being active in `#portuguese` is a plus
+- you are online and available to moderate and react to reports in the Australia/Asia timezones
+- you actively browse and/or participate in forum discussions
+
+---
+
+If you fit the above-mentioned criteria, **click here to apply!** Feel free to take your time when answering the questions, but don't take too long as the applications will remain **open for 2 weeks**, closing on **March XX, 2025 23:59 UTC+0**.
```

Comment:
pr desc checkbox is nice but i recommend making it a placeholder link so ci fails and blocks the pr too

Reply:
https://github.com/ppy/osu-wiki/pull/12928/commits/a166066e22b4b2cdfc3f5d55c8f424ffeaf93eb5